index.js

```diff
@@ -193,6 +193,9 @@ const AvatarEditor = React.createClass({
   },
 
   isDataURL (str) {
+    if (str === null) {
```

Comment:
IMHO, it's not the function's responsibility to check whether the provided param is null.